ports/orc/portfile.cmake

```
@@ -16,21 +16,21 @@ file(REMOVE "${SOURCE_PATH}/cmake_modules/FindProtobuf.cmake")
 file(REMOVE "${SOURCE_PATH}/cmake_modules/FindSnappy.cmake")
 file(REMOVE "${SOURCE_PATH}/cmake_modules/FindZLIB.cmake")
 
-if(CMAKE_HOST_WIN32)
-  set(PROTOBUF_EXECUTABLE ${CURRENT_INSTALLED_DIR}/tools/protobuf/protoc.exe)
-else()
-  set(PROTOBUF_EXECUTABLE ${CURRENT_INSTALLED_DIR}/tools/protobuf/protoc)
-endif()
+set(PROTOBUF_EXECUTABLE "${CURRENT_HOST_INSTALLED_DIR}/tools/protobuf/protoc${VCPKG_HOST_EXECUTABLE_SUFFIX}")
 
 if(VCPKG_TARGET_IS_WINDOWS)
   set(BUILD_TOOLS OFF)
 else()
   set(BUILD_TOOLS ON)
 endif()
 
+if(NOT VCPKG_TARGET_IS_UWP)
+    set(configure_opts PREFER_NINJA)
+endif()
+
 vcpkg_configure_cmake(
```

```suggestion
vcpkg_cmake_configure(
```
Has the port a problem with the new one?

---

And this should really be `vcpkg_cmake_configure` since you removed `PREFER_NINJA`

---

I didn't remove it.... It is just not added for UWP. If you want this changed please open up a PR against my fork so I can merge it. I really don't have time to update all those small details.

---

Have a look at 2 lines below from the files view. The diff shows that you replaced `PREFER_NINJA` with `${configure_opts}`.  But I can open a PR anyway.